### PR TITLE
style: move determiners out of link texts

### PR DESCRIPTION
**Pull request by roj1512**

### CODE REVIEW

**Review by KnorpelSenf — APPROVED** [review]
Excellent

File: site/docs/advanced/deployment.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ This depends on your deployment type.
 ### Webhooks
 
 1. Make sure you do not perform any long-running operations in your middleware, such as large file transfers.
-   [This leads to timeout errors](../guide/deployment-types.md#ending-webhook-requests-in-time) for the webhooks, and duplicate update processing as Telegram will re-send non-acknowledged updates.
+   This leads to [timeout errors](../guide/deployment-types.md#ending-webhook-requests-in-time) for the webhooks, and duplicate update processing as Telegram will re-send non-acknowledged updates.
```
Comment (KnorpelSenf) [inline site/docs/advanced/deployment.md:39]: I'd revert this one. The link is about how one thing leads to another, which is what the section explains. It's not about timeout errors only.

File: site/docs/guide/introduction.md [hunk 2]
```diff
@@ -87,7 +87,7 @@ If you see unfamiliar syntax in the docs, or if you get an error message that yo
 :::
 
 ::: danger Not Learning How to Code
-Save yourself some time by watching [this 34 second long video](https://youtu.be/8RtGlWmXGhA).
+Save yourself some time by watching this [34-second-long video](https://youtu.be/8RtGlWmXGhA).
```
Comment (KnorpelSenf) [inline site/docs/guide/introduction.md:90]: Would this be better?
```suggestion
Save yourself some time by watching this [34-second video](https://youtu.be/8RtGlWmXGhA).
```

**Review by KnorpelSenf — APPROVED** [review]

File: site/docs/guide/introduction.md (hunk 2 above)
Comment (roj1512) [reply site/docs/guide/introduction.md:90]: Definitely!

**Pull request merged**